boot.php

```diff
@@ -51,51 +44,6 @@
  */
 define('EOL',                    "<br />\r\n");
 
-/**
- * Image storage quality.
- *
- * Lower numbers save space at cost of image detail.
- * For ease of upgrade, please do not change here. Set system.jpegquality = n in config/local.config.php,
- * where n is between 1 and 100, and with very poor results below about 50
- */
-define('JPEG_QUALITY',            100);
-
-/**
- * system.png_quality = n where is between 0 (uncompressed) to 9
- */
-define('PNG_QUALITY',             8);
-
-/**
- * An alternate way of limiting picture upload sizes. Specify the maximum pixel
- * length that pictures are allowed to be (for non-square pictures, it will apply
- * to the longest side). Pictures longer than this length will be resized to be
- * this length (on the longest side, the other side will be scaled appropriately).
- * Modify this value using
- *
- * 'system' => [
- *      'max_image_length' => 'n',
- *      ...
- * ],
- *
- * in config/local.config.php
- *
- * If you don't want to set a maximum length, set to -1. The default value is
- * defined by 'MAX_IMAGE_LENGTH' below.
- */
-define('MAX_IMAGE_LENGTH',        -1);
-
-/**
- * Not yet used
- */
-define('DEFAULT_DB_ENGINE',  'InnoDB');
-
-/** @deprecated since version 2019.03, please use \Friendica\Module\Register::CLOSED instead */
-define('REGISTER_CLOSED',        \Friendica\Module\Register::CLOSED);
-/** @deprecated since version 2019.03, please use \Friendica\Module\Register::APPROVE instead */
-define('REGISTER_APPROVE',       \Friendica\Module\Register::APPROVE);
-/** @deprecated since version 2019.03, please use \Friendica\Module\Register::OPEN instead */
-define('REGISTER_OPEN',          \Friendica\Module\Register::OPEN);
```

Comment:
How old? Are we allowing direct updates from a version with that configuration file format?

Reply:
There's little we can prevent in this case, but you're making a good point. Please proceed.

Reply:
Is there something that needs to be changed in the config file?  I think I have a line for the register policy in my config file. And I haven't touched that line for years.

Reply:
What's the value in that line?

Reply:
`'register_policy' => \Friendica\Module\Register::APPROVE,`

Reply:
That's totally fine.

Reply:
Thanks. I guess I could also remove that line fin the config file because the policy is configured via admin panel and the value is stored in the database?

Reply:
Just leave it as it is. It doesn't hurt.